session/session.go

```diff
@@ -189,11 +190,11 @@ func (s *session) handleRestoredCh(pch *pclient.Channel) {
 	if pch.Phase() != pchannel.Acting {
 		return
 	}
-	peers := pch.Peers()
-	parts := make([]perun.Peer, len(peers))
-	aliases := make([]string, len(peers))
+	peerIDs := pch.Peers()
```

Comment:
```suggestion
	partOffChainAddrs := pch.Peers()
```

The data returned by `pch.Peers()` is indeed the off-chain addresses of the channel participants. Hence I think the suggested name is more descriptive.